defaults/clog.h

```diff
@@ -42,6 +42,10 @@
 #define CLOG_STRUCT(numFields) CLOG_STRUCT_HELPER(numFields)
 #endif
 
+#define CLOG_ARRAY_HELPER(length, pointer) pointer, length
+#define CLOG_ARRAY(length, pointer) CLOG_ARRAY_HELPER(length, pointer)
+#define CLOG_STRUCT_HELPER(numFields) numFields
+#define CLOG_STRUCT(numFields) CLOG_STRUCT_HELPER(numFields)
```

Comment:
Why? This is already defined above, no?

Reply:
agreed;  I removed some of these - they're there as part of another branch and in this context they're confusing - we'll address them when merging the other branch later on